portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,14 +47,21 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(messages);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    String message = messages.get((int) (Math.random() * messages.size()));
+    response.setContentType("text/html;");
+    response.getWriter().println(message);
   }
 
-  private String convertToJson(ArrayList<String> messages){
-      Gson gson = new Gson();
-      String json = gson.toJson(messages);
-      return json;
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Same. Can you make this more descriptive?

Reply:
Same as above. Added a description comment